src/components/blocks/image-carosel/_image-carosel.js

```diff
@@ -0,0 +1,135 @@
+(function() {
+	if (!!document.querySelector('.image-carosel')) {
```

Comment:
Can we do a rename throughout to correct the spelling to carousel

Reply:
renamed to carousel throughout